build-system/release-workflows/archive-release.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const fastGlob = require('fast-glob');
+const fs = require('fs-extra');
+const path = require('path');
+const {cyan} = require('kleur/colors');
+const {log} = require('../common/logging');
+const {runCiJob} = require('./ci-job');
+const {timedExecOrDie} = require('../pr-check/utils');
+
+/**
+ * @fileoverview Script that builds a release.
+ */
+
+const jobName = 'archive-release.js';
+
+const SRCS_DIR = '/tmp/restored-workspace/releases';
+const DEST_DIR = '/tmp/release';
+const ARTIFACT_FILE_NAME = '/tmp/release.tar.gz';
+
+/**
+ * Merge the outputs of the flavor (both module/nomodule) into the dest dir.
+ * @param {string} flavor
+ */
+function mergeOutputs_(flavor) {
+  for (const esm of fs.readdirSync(path.join(SRCS_DIR, flavor))) {
+    log('Merging', cyan(flavor), 'flavor', cyan(esm), 'build');
+    fs.copySync(path.join(SRCS_DIR, flavor, esm), DEST_DIR, {
+      overwrite: true,
+      filter: (src) => !src.endsWith('/files.txt'),
+    });
+  }
+}
+
+/**
+ * Merge the files.txt files for the flavor's module/nomodule outputs.
+ * @param {string} flavor
+ */
+function mergeFilesTxt_(flavor) {
+  const /** @type Map<string, Set<string>> */ filesByRtv = new Map();
+
+  for (const filesTxtPath of fastGlob.sync(
+    path.join(SRCS_DIR, flavor, '*/org-cdn/rtv/*/files.txt')
+  )) {
```

Comment:
optional nit: For readability / debuggability, maybe create a variable `filesTxtPaths` with the result of `fastGlob` and then iterate on it?

Reply:
Done.